### ISSUE TITLE
JSON issue preventing the adding of contacts

### ISSUE THREAD

**Issue by awesomebjt** [JohnRillos/whom#1]
When I attempt to add a contact or import pals, errors like this appear in my terminal and the action fails in the UI.

```clay: read-at-tako fail [desk=%whom care=%c case=[%da p=~2023.5.21..00.18.42..4845] path=/json/whom-action]
-find.$
/mar/json/hoon::[11 9].[11 13]>
/mar/json/hoon::[11 5].[11 13]>
/mar/json/hoon::[11 1].[26 3]>
/mar/json/hoon::[10 1].[26 3]>
/mar/json/hoon::[9 1].[26 3]>
/mar/json/hoon::[8 1].[26 3]>
[%error-building /mar/json/hoon]
[%error-building-cast %json %whom-action]
[%error-building-tube %json %whom-action]
```

I've heard that with zuse 413 there were breaking changes to json parsing. I'm afraid I don't know much more than that but this issue was happening on my planet, and I was able to reproduce it on  a fresh comet, so I'm pretty sure it isn't due to any corrupted desks on my planet.

**Comment by JohnRillos** [comment]
Thanks for reporting this, I'll take a look. It probably is due to the 413
change.

**Comment by JohnRillos** [comment]
JSON parsing is fixed now in %whom `1.9.1`. Sorry about that, let me know if you run into any other issues!

Master: https://github.com/JohnRillos/whom/commit/aa91c144f39503570db1d68626a1557aa2042976
Build: https://github.com/JohnRillos/whom/commit/c9b255c6cfc66df839fbb23751b35ed277aac929

**Comment by awesomebjt** [comment]
That was fast, well done! Seems to be working now.
